### ISSUE TITLE
Failures not showing up on Update Tests - Details

### ISSUE THREAD

**Issue by KaiRo-at** [mozilla/mozmill-dashboard#134]
On http://mozmill-release.blargon7.com/#/update/detail?&channel=beta&from=2014-12-05&to=2014-12-05&target=35.0 there is all green, whereas we had a ton of failures on that day on Mac and Windows that show up correctly in "Update Tests - Reports", see 
http://mozmill-release.blargon7.com/#/update/reports?app=All&branch=All&platform=Mac&from=2014-12-03&to=2014-12-06 and http://mozmill-release.blargon7.com/#/update/reports?app=All&branch=All&platform=Win&from=2014-12-03&to=2014-12-06

Examples of run that should show up:
http://mozmill-release.blargon7.com/#/update/report/eaa287eb6543546f0b0d98e1f0505eca (but mo "ast" even shows up for 32 on Mac in the Details dashboard)
http://mozmill-release.blargon7.com/#/update/report/eaa287eb6543546f0b0d98e1f04faccb (but no "sq" shows up for 31 on Windows)

**Comment by KaiRo-at** [comment]
Interestingly, it looks like I see failures in recent queries. Did something change?

See e.g. http://mozmill-release.blargon7.com/#/update/detail?&channel=beta&from=2015-01-08&to=2015-01-08&target=35.0

**Comment by whimboo** [comment]
No, nothing has been changed. Please read my last comment carefully enough. The reason why you see those failures is that the updates were successful but the last test for the updated version of Firefox failed due to the buildid. And as said we can only query for all the update tests of the target build, means if an update failed we do not see it.

**Comment by KaiRo-at** [comment]
Ah, ok, right, we get the right version on those failures, just the wrong build ID, so it shows up for this version correctly, I realize that now. :)
So, when we don't even get to the right version, I guess we end up just not displaying it on this view, which is the bug.

**Comment by whimboo** [comment]
That's totally correct. That's why better use the update overview page with the right date and channel selection.
